Example/ViewController.swift

```diff
@@ -239,13 +236,15 @@ class ViewController: UIViewController {
     func startBasicNavigation() {
         guard let response = response, let route = response.routes?.first, case let .route(routeOptions) = response.options else { return }
         
+        let waypoints = routeOptions.waypoints
```

Comment:
I don't think we need `let waypoints = routeOptions.waypoints` here.

Reply:
Deleted